projects/storefrontlib/cms-structure/services/cms-features.service.spec.ts

```diff
@@ -15,7 +15,7 @@ const mockCmsConfig: CmsConfig = {
       module: async () => MockFeature1Module,
```

Comment:
How did this change appear? You noticed something that we missed to carry-over from the epic/cart?